### DISCUSSION TITLE
Currencies

### DISCUSSION THREAD

**Discussion by richardbirkin** [SuperHyperInstantFutureTime/TrackShift#179] [General]
@g105b asked some questions about distribution statement currencies where the currency is not included in the data.

CD Baby = USD

<img width="709" alt="Screenshot 2024-06-20 at 11 06 50" src="https://github.com/SuperHyperInstantFutureTime/TrackShift/assets/115098450/ac07273b-b341-4113-a99e-9eedf05a2f90">

Cargo Physical = GBP 
as they are a UK company. 
Although this should be double checked with Cargo - which I will do ASAP. For now, assume GBP as default.

**Comment by g105b** [comment]
For Cargo Digital, the currency is specified in the upload data... it was just Cargo Physical that I was unable to verify.
